image_writers_guide/guidelines.adoc

```diff
@@ -5,143 +5,186 @@
 :icons:
 :experimental:
 :toc: macro
-:toc-title: 
+:toc-title:
 
 toc::[]
 
 == Overview
-When creating Docker images to run on OpenShift there are a number of best practices to consider in order to ensure consumers of those images have a good experience.  Since images are intended to be immutable and used as is, it is the image author's responsibility to ensure the image contains any artifacts that would typically be needed by a user of the image and that the image can be configured by the user.  Here are some general comments around things to consider.
+When creating Docker images to run on OpenShift, there are a number of best practices to consider as an image author to ensure a good experience for consumers of those images. Because images are intended to be immutable and used as is, it is your responsibility to ensure:
 
-== General Docker Best Practices
-The following are some guidelines that apply to Docker image creation in general, independent of whether the images are used on OpenShift v3.
+- the image contains any artifacts that would typically be needed by a user of the image, and
+- the image can be configured by the user.
```

Comment:
I agree, I don't think the bullet points are necessary, even if you wanted to keep the content.